const core = require('@actions/core')
core.debug('Inside try block');